README.md

```diff
@@ -5,18 +5,14 @@ boilerplates.
 
 ## Important Notice
 
-:warning: Please be aware that [templates.netlify.com](https://templates.netlify.com/) pages will be redirected to relevant GitHub repos, [jamstack.org/](https://www.jamstack.org/), or [www.netlify.com](https://www.netlify.com/) pages. If you are interested in adding a template to [netlify.com/integrations/templates/](https://www.netlify.com/integrations/templates/), please reach out to templates@netlify.com
-
-
-
+:warning: [templates.netlify.com](https://templates.netlify.com/) is now being redirected to relevant GitHub repos, [jamstack.org/](https://www.jamstack.org/), or [www.netlify.com](https://www.netlify.com/) pages. If you are interested in adding a template to [netlify.com/integrations/templates/](https://www.netlify.com/integrations/templates/), please reach out to templates@netlify.com
```

Comment:
@hexagoncircle smol request to update the link for jamstack.org to `(https://jamstack.org/)`